tautulli/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: tautulli
```

Comment:
this should `tautulli_web_1`, since we are letting Docker Compose auto-name containers